apps/backend/src/database/migrations/1690503718109-daily-date.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1690503718109 implements MigrationInterface {
```

Comment:
Overall suggestion: change all these to `parsed_date` for each table, do not do it for `file_uploaded` entity